store/mongo/datastore_mongo.go

```diff
@@ -264,7 +265,8 @@ var (
 	ErrDeploymentStorageCannotExecQuery   = errors.New("Cannot execute query")
 	ErrStorageInvalidInput                = errors.New("invalid input")
 
-	ErrLimitNotFound = errors.New("limit not found")
+	ErrLimitNotFound      = errors.New("limit not found")
```

Comment:
Is it used anywhere?

Reply:
Actually it's not. It was also my concern, but thought maybe it's intentional fur future purposes.

Reply:
You know, actually it's in use.